tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -95,22 +99,45 @@ def test_streams_runs_with_broker_down_initially(self):
         self.wait_for_verification(processor_2, broker_unavailable_message, processor_2.LOG_FILE, 10)
         self.wait_for_verification(processor_3, broker_unavailable_message, processor_3.LOG_FILE, 10)
 
-        # now start broker
-        self.kafka.start_node(node)
-
-        # assert streams can process when starting with broker down
-        self.assert_produce_consume(self.inputTopic,
-                                    self.outputTopic,
-                                    self.client_id,
-                                    "running_with_broker_down_initially",
-                                    num_messages=9,
-                                    timeout_sec=120)
-
-        message = "processed3messages"
-        # need to show all 3 instances processed messages
-        self.wait_for_verification(processor, message, processor.STDOUT_FILE)
-        self.wait_for_verification(processor_2, message, processor_2.STDOUT_FILE)
-        self.wait_for_verification(processor_3, message, processor_3.STDOUT_FILE)
+        with processor.node.account.monitor_log(processor.LOG_FILE) as monitor_1:
+            with processor_2.node.account.monitor_log(processor_2.LOG_FILE) as monitor_2:
+                with processor_3.node.account.monitor_log(processor_3.LOG_FILE) as monitor_3:
+                    self.kafka.start_node(node)
+
+                    monitor_1.wait_until(self.connected_message,
+                                         timeout_sec=120,
+                                         err_msg=("Never saw '%s' on " % self.connected_message) + str(processor.node.account))
+                    monitor_2.wait_until(self.connected_message,
+                                         timeout_sec=120,
+                                         err_msg=("Never saw '%s' on " % self.connected_message) + str(processor_2.node.account))
+                    monitor_3.wait_until(self.connected_message,
+                                         timeout_sec=120,
+                                         err_msg=("Never saw '%s' on " % self.connected_message) + str(processor_3.node.account))
+
+        with processor.node.account.monitor_log(processor.STDOUT_FILE) as monitor_1:
+            with processor_2.node.account.monitor_log(processor_2.STDOUT_FILE) as monitor_2:
+                with processor_3.node.account.monitor_log(processor_3.STDOUT_FILE) as monitor_3:
+
+                    self.assert_produce(self.inputTopic,
```

Comment:
break assertion of production to streams app and consuming from topic written to by streams into separate parts. I've done this in other parts of the test, but I won't comment on each section.